app/Modules/Group/Events/Traits/IsPublishableApplicationEvent.php

```diff
@@ -18,27 +18,27 @@ public function getEventType(): string
     }
 
     public function getPublishableMessage(): array
-{
-    return [
-        'expert_panel' => array_merge([
-            'id' => $this->group->uuid,
-            'long_name' => $this->group->name,
-            'short_name' => $this->group->expertPanel->short_base_name,
-            'status' => optional($this->group->groupStatus)->name, // Retrieve the status name
-            'group updated date' => $this->group->updated_at,
-            'parent_group' => optional($this->group->parentGroup)->name,
-            'type' => $this->group->fullType->name,
-            'affiliation_id' => $this->group->expertPanel->affiliation_id,
-            'scope description' => $this->group->expertPanel->scope_description,
-        ],
-        // Conditionally add vcep fields below if type is 'vcep'
-        $this->group->fullType->name === 'vcep' ? ['clinvar_id' => null] : [],
-        $this->group->fullType->name === 'vcep' ? ['clinvar_url' => null] : [],
-        $this->group->fullType->name === 'vcep' ? ['cspec_url' => $this->group->expertPanel->affiliation_id] : [],
-        $this->group->fullType->name === 'vcep' ? ['vspec_web_address' => null] : [],
-        )
-    ];
-}
+    {
+        return [
+            'expert_panel' => array_merge(
+                [
+                    'id' => $this->group->uuid,
+                    'name' => $this->group->name,
+                    'short_name' => $this->group->expertPanel->short_base_name,
+                    'status' => optional($this->group->groupStatus)->name, // Retrieve the status name
+                    'parent_group' => optional($this->group->parentGroup)->name,
+                    'type' => $this->group->fullType->name,
+                    'affiliation_id' => $this->group->expertPanel->affiliation_id,
+                    'scope_description' => $this->group->expertPanel->scope_description,
+                ],
+                // Conditionally add vcep fields below if type is 'vcep'
+                $this->group->fullType->name === 'vcep' ? ['clinvar_id' => null] : [],
```

Comment:
I am hesitant to merge to main while these vcep fields (clinvar_id and clinvar_url) remain null as placeholders.